content/en/docs/concepts/storage/volumes.md

```diff
@@ -1017,6 +1017,42 @@ spec:
         claimName: my-lamp-site-data
 ```
 
+### Using subPath with expanded environment variables
+
+{{< feature-state for_k8s_version="v1.11" state="alpha" >}}
+
+
+Before you use this feature, you must enable the `VolumeSubpathEnvExpansion`feature gate.
```

Comment:
Ah sorry, there were actually 2 sentences, and I was only talking about removing the first.  Could you add this back?
```
`subPath` directory names can also be constructed from Downward API environment variables.
```

Reply:
@msau42 Put back the descriptive sentence